SMXLoadingImage.ios.js

```diff
@@ -40,13 +40,13 @@ var SMXLoadingImage = React.createClass({
     };
   },
 
-  setNativeProps(props) {
+  setNativeProps(props:Object) {
 
   },
 
   _animate: function () {
     this.state.angle.setValue(0);
-    this._anim = Animated.timing(this.state.angle, {
```

Comment:
This could also be fixed by add a default value for _anim to the createClass call, but since _anim isn't used elsewhere  thought it easier to delete it.